From abf2fb34a5754ff08cecace810a1d1c095b8c136 Mon Sep 17 00:00:00 2001
From: Liang Chen <cl@rock-chips.com>
Date: Thu, 27 Jun 2019 21:19:39 +0800
Subject: [PATCH] arm64: dts: rockchip: limit vdd_npu to 0.88V for
 rk3399pro-npu

The recommended max voltage of vdd_npu is 0.88V in the datasheet.

Change-Id: I9713810692c5d32b8c41b0b0e0d02405c01dd0b7
Signed-off-by: Liang Chen <cl@rock-chips.com>
---
 .../arm64/boot/dts/rockchip/rk3399pro-npu.dtsi | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
index e98fe9aeff10..3753f1f95e45 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399pro-npu.dtsi
@@ -587,7 +587,7 @@
 	npu_opp_table: npu-opp-table {
 		compatible = "operating-points-v2";
 
-		rockchip,max-volt = <950000>;
+		rockchip,max-volt = <880000>;
 		rockchip,evb-irdrop = <37500>;
 
 		rockchip,pvtm-voltage-sel = <
@@ -607,26 +607,26 @@
 
 		opp-200000000 {
 			opp-hz = /bits/ 64 <200000000>;
-			opp-microvolt = <750000 750000 950000>;
+			opp-microvolt = <750000 750000 880000>;
 		};
 		opp-297000000 {
 			opp-hz = /bits/ 64 <297000000>;
-			opp-microvolt = <750000 750000 950000>;
+			opp-microvolt = <750000 750000 880000>;
 		};
 		opp-400000000 {
 			opp-hz = /bits/ 64 <400000000>;
-			opp-microvolt = <750000 750000 950000>;
+			opp-microvolt = <750000 750000 880000>;
 		};
 		opp-594000000 {
 			opp-hz = /bits/ 64 <594000000>;
-			opp-microvolt = <750000 750000 950000>;
+			opp-microvolt = <750000 750000 880000>;
 		};
 		opp-792000000 {
 			opp-hz = /bits/ 64 <792000000>;
-			opp-microvolt = <825000 825000 950000>;
-			opp-microvolt-L0 = <825000 825000 950000>;
-			opp-microvolt-L1 = <800000 800000 950000>;
-			opp-microvolt-L2 = <775000 775000 950000>;
+			opp-microvolt = <825000 825000 880000>;
+			opp-microvolt-L0 = <825000 825000 880000>;
+			opp-microvolt-L1 = <800000 800000 880000>;
+			opp-microvolt-L2 = <775000 775000 880000>;
 		};
 	};
 
-- 
2.35.3

